.travis.yml

```diff
@@ -23,11 +23,11 @@ env:
   - INSTANCE=chefdk-debian-9
   - INSTANCE=chefdk-centos-6
   - INSTANCE=chefdk-centos-7
-  - INSTANCE=chefdk-centos-7 CHEF_VERSION=12.7.2
+  - INSTANCE=chefdk-centos-7 CHEF_VERSION=13.12.3
```

Comment:
It's probably better to set this to 13.1 so we test an early Chef 13 release.

Reply:
sure can do that